docs/secure-connections/sauce-connect/quickstart.md

```diff
@@ -46,58 +46,43 @@ Get up and running with a Sauce Connect Proxy tunnel in just a few minutes by fo
 
   </TabItem>
   </Tabs>
-4. Log in to Sauce Labs.
-5. Go to the [**TUNNELS**](https://app.saucelabs.com/tunnels) page > Scroll to **STEP 3: Configure & Authenticate** > Copy the code snippet.
-6. Paste the snippet into your local terminal. You can re-name your tunnel or leave it as-is.
+3. Log in to Sauce Labs.
+4. Go to the [**TUNNELS**](https://app.saucelabs.com/tunnels) page.<br/><img src={useBaseUrl('img/sauce-connect/tunnelsPage.png')} alt="Sauce Connect Proxy Tunnels page" width="250"/>
+5. Skip to **STEP 3: Configure & Authenticate** and copy the code snippet.<br/><img src={useBaseUrl('img/sauce-connect/configureAuth.png')} alt="Sauce Connect Proxy Tunnels page code snippet" width="500"/>
+6. Paste the entire snippet into your local terminal. Optionally, you can re-name your tunnel using the `--tunnel-name` flag.
+  ```bash
+  ./sc -u {SAUCE_USERNAME} -k {SAUCE_ACCESS_KEY} --region us-west --tunnel-name {TUNNEL_NAME}
```

Comment:
I am getting an error that says `--region` is an unknown flag? Is this a new config and I am using an old version of SC?

Reply:
Yes that's probably it! The `./sc` and `--tunnel-name` parts are 4.7.0+ specific, too.